modules/op-resolver-stepactions-ref.adoc

```diff
@@ -0,0 +1,79 @@
+  steps:
+  - name: clone-repo-anon-step
+    ref:
+      resolver: http
+      params:
+      - name: url
+        value: https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml
+    params:
+    - name: URL
+      value: $(params.url)
+    - name: OUTPUT_PATH
+      value: $(workspaces.output.path)
+----
+
+.Supported parameters for the `git-clone` step action
+[cols="1,2,1,1",options="header"]
+|===
+| Parameter | Description | Type | Default value
+|`OUTPUT_PATH` | A directory for the fetched Git repository. Cloned repo data is placed in the root of the directory or in the relative path defined by the `SUBDIRECTORY` parameter |`string` |
+|`SSH_DIRECTORY_PATH` |A `.ssh` directory with the private key, `known_hosts`, `config`, and other files as necessary. If you provide this directory, the task uses it for authentication to the Git repository. Bind the workspace providing this directory to a `Secret` resource for secure storage of authentication information. |`string` |
+|`BASIC_AUTH_PATH` |A directory containing a `.gitconfig` and `.git-credentials` files. If you provide this directgory, the task uses it for authentication to the Git repository. Use a `SSH_DIRECTORY_PATH` directory for authentication instead of `BASIC_AUTH_PATH` whenever possible. Bind the workspace providing this directory to a `Secret` resource for secure storage of authentication information. |`string` |
+|`SSL_CA_DIRECTORY_PATH` |A workspace containing CA certificates. If you provide this workspace, Git uses these certificates to verify the peer when interacting with remote repositories using HTTPS. |`string` |
+|`CRT_FILENAME` |Certificate Authority (CA) bundle filename in the `ssl-ca-directory` workspace. |`string` |`ca-bundle.crt`
+|`HTTP_PROXY` |HTTP proxy server (non-TLS requests). |`string` |
+|`HTTPS_PROXY` |HTTPS proxy server (TLS requests). |`string` |
+|`NO_PROXY` |Opt out of proxying HTTP/HTTPS requests. |`string` |
+|`SUBDIRECTORY` |Relative path in the `output` workspace where the task places the Git repository. |`string` |
+|`USER_HOME` |Absolute path to the Git user home directory in the pod. |`string` |`/home/git`
+|`DELETE_EXISTING` |Delete the contents of the default workspace, if they exist, before running the Git operations. |`string` |`true`
+|`VERBOSE` |Log the executed commands. |`string` |`false`
+|`SSL_VERIFY` |The global `http.sslVerify` value. Do not set this parameter to to `false` unless you trust the remote repository. |`string` |`true`
+|`URL` |Git repository URL. |`string` |
+|`REVISION` |The revision to check out, for example, a branch or tag. |`string` |`main`
+|`REFSPEC` |The `refspec` string for the repository that the task fetches before checking out the revision. |`string` |
+|`SUBMODULES` |Initialize and fetch Git submodules. |`string` |`true`
+|`DEPTH` |Number of commits to fetch, a "shallow clone" is a single commit. |`string` |`1`
```

Comment:
The way these two last entries are worded look a little off to me, probably it's the quotes. Do you think they are necessary? Is "shallow clone" supposed to mean something to some people? Also perhaps this entry could be broken up into two sentences?

Though now I'm seeing that this wording might already be used elsewhere. So it's fine to keep as is, but just something to consider in the future.